pages/common/quarto.md

```diff
@@ -0,0 +1,36 @@
+# quarto
+
+> An open-source scientific and technical publishing system built on Pandoc.
+> More information: <https://quarto.org/>.
+
+- Create a Quarto project:
+
+`quarto create-project {{destination_directory}}`
+
+- Create a new website project:
+
+`quarto create-project {{destination_directory}} --type website`
```

Comment:
```suggestion
`quarto create-project {{destination_directory}} --type {{website}}`
```
Even tough it is a specific example, it still is an argument. This applies to all `--type` and `--to` flags.